### ISSUE TITLE
table-help

### ISSUE THREAD

**Issue by andrejsharapov** [Templarian/MaterialDesign#6586]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

My team and I are building tables for clients with complex functionality and have already created instructions for working with them. In the navigation bar, we are currently using a regular table icon, but on the page with tables, we would like to display not just a question mark or a table icon, but both together, in one icon.

### Share a preview of your contribution.

![image](https://user-images.githubusercontent.com/44722574/167927398-142ff340-8cc4-49e9-8185-4e9e80c99989.png)

### Upload the icon source.

[table-help.zip](https://github.com/Templarian/MaterialDesign/files/8672782/table-help.zip)

**Comment by goyney** [comment]
We've never used the cutout for a help variant before, so we need to figure out the pattern we would want to follow from this point on. I'd like to see this with a 2dp cutout as well so we can evaluate both options.

**Comment by andrejsharapov** [comment]
| circle    | cut 2dp |
| --------- | ------- |
| ![circle] | ![cut]  |

<details>
<summary>I had to make the question a little lower so that it overlaps the bottom edge of the table</summary>

![image](https://user-images.githubusercontent.com/44722574/168884742-519ef856-098f-40cd-bc51-ed077a34a63e.png)


</details>

[circle]: https://user-images.githubusercontent.com/44722574/168884393-c742bfb9-2eda-4bb2-93bb-93de6755938a.png
[cut]: https://user-images.githubusercontent.com/44722574/168883912-6da3443d-fbb9-4c2c-9d0b-a0ca497ad02d.png

**Comment by Templarian** [comment]
The 2dp cutout on the question mark might be the best.

**Comment by andrejsharapov** [comment]
I tried three different sizes for the question with  cutout and this is what came out :roll_eyes:

v1
![image](https://user-images.githubusercontent.com/44722574/169389798-843de50e-2732-4427-96d4-1b948efa5adf.png)


v2a, v2b
![image](https://user-images.githubusercontent.com/44722574/169386742-a70e4839-e236-4945-94cd-160ecd621421.png) ![image](https://user-images.githubusercontent.com/44722574/169386632-b1e118a1-5e29-4859-a45e-821035c1c52a.png)  

v3
![image](https://user-images.githubusercontent.com/44722574/169386950-4ebd182a-6944-4356-b309-1850ca5ab66f.png)

**Comment by andrejsharapov** [comment]
I also noticed that many of the icons use different spellings of the question mark.

**Comment by goyney** [comment]
> I also noticed that many of the added icons use different spellings of the question mark.

Yeah, this is a very good catch. Our official modifier name is `question`. Anything using `help` should be renamed. I will capture these in another issue and slate them for our v7 release. 👍🏼

**Comment by goyney** [comment]
[![table-question](https://materialdesignicons.com/icon/table-question)](https://materialdesignicons.com/icon/table-question)
